.github/workflows/promote.yml

```diff
@@ -90,6 +90,15 @@ jobs:
             --sha "$(git rev-parse --short HEAD)" \
             --channel ${{ inputs.env }} --no-xz --indexes
 
+      - name: Promote cli as latest
```

Comment:
here latest, there stable 💭 

I wonder if we can unify it 🤔 

but for now lgtm